bcl2fastq/lib/bcl2fastq_utils.py

```diff
@@ -41,9 +45,16 @@ def __init__(self,
         """
 
         self.runfolder_input = runfolder_input
-        self.samplesheet_file = runfolder_input + "/SampleSheet.csv"
         self.base_calls_input = runfolder_input + "/Data/Intensities/BaseCalls"
 
+        if not samplesheet:
```

Comment:
All right. I'm mainly thinking for things like integrating with Stackstorm easier, so we notice what/when something weird gets sent in. But if you think this is enough I'll go ahead and merge.